spec/factories/group.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :group do |f|
+    f.name 'group'
+    f.description 'this is a group'
```

Comment:
I see from your group test that names should be unique - should `name` be a sequence then? 

Also, you can define factories without the block variable, e.g.

```ruby
FactoryGirl.define do
  factory :group do
    sequence(:name){ |n| "Group #{n}" }
    description 'Description'
  end
end
```